src/python_testing/TC_RVCRUNM_2_1.py

```diff
@@ -90,15 +90,25 @@ async def test_TC_RVCRUNM_2_1(self):
         if self.is_ci:
             app_pid = self.matter_test_config.app_pid
             if app_pid == 0:
-                asserts.fail("The --app-pid flag must be set when PICS_SDK_CI_ONLY is set.c")
-            self.app_pipe = self.app_pipe + str(app_pid)
```

Comment:
this may be needed.

Reply:
Thank you, you are correct that was definitely needed in there. I have returned it to its rightful place here now.